lib/legacy/static_data.rb

```diff
@@ -40,12 +40,9 @@ def self.populate
       loan_type = OptionSet.find_or_create_by(division: ::Division.root, model_type: ::Loan.name, model_attribute: 'loan_type')
       loan_type.options.destroy_all
 
-      # Note, there is currently no business logic dependency on these options, so no need for a 'slug' style value.
+      # Note, there is currently no business logic dependency on these options, # so no need for a 'slug' style value.
```

Comment:
Was a line break intended here? Right now there is just an extra `#` in the comment.